cmd/agent/main.go

```diff
@@ -1,3 +1,143 @@
 package main
 
-func main() {}
+import (
+	"flag"
+	"fmt"
+	"math/rand"
+	"os"
+	"runtime"
+	"strconv"
+	"time"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/monsterr00/metric-service.gittest_client/internal/config"
+)
+
+var gauge map[string]float64
+var counter map[string]int64
+
+func sendToServer(client *resty.Client) {
+	for k, v := range gauge {
+		metricGaugeURL := fmt.Sprintf("/update/gauge/%s/%f", k, v)
+		requestURL := fmt.Sprintf("%s%s%s", "http://", config.ClientOptions.Host, metricGaugeURL)
+
+		req, err := client.R().
+			SetHeader("Content-Type", "text/plain").
+			Post(requestURL)
+		if err != nil {
+			fmt.Printf("Client: error sending http-request: %s\n", err)
+			os.Exit(1)
```

Comment:
CHANGE REQUEST: you shouldn't panic in the middle of the program.

Reply:
done, has removed panics